### ISSUE TITLE
关于公式中的参数lamda

### ISSUE THREAD

**Issue by HZQ-GitHub** [hsmyy/CenterLoss_Caffe_Mnist#8]
center_loss的公式中有一个参数lamda,即lamda/2*sum(f(x,c)),请问,参数lamda对应于代码中那个变量呢?
您代码中似乎没有出现这个参数哦.
 loss += caffe_cpu_dot(channels, center_loss_.cpu_data() + i * channels,
center_loss_.cpu_data() + i*channels) / Dtype(2.0) / static_cast<Dtype>(num)
另外,这个参数应该设置为多少呢,谢谢

**Comment by allenxcp** [comment]
lamda= loss_weight
